app/Http/Controllers/UserController.php

```diff
@@ -6,15 +6,18 @@
 use App\Models\User;
 use App\Http\Requests\UserRequest;
 use Illuminate\Support\Facades\Hash;
+use App\Services\UserService;
 
 class UserController extends Controller
 {
+    protected $userService;
+    public function __construct(UserService $userService)
```

Comment:
remove row 13.
update row 14 to: public function __construct(protected UserService $userService)